### PR TITLE
WIP: Geo feature

### PR DESCRIPTION
**Pull request by kjoscha**
@0x46616c6b This is the state so far. If you find some time, feel free to review. We wait for the backend implementation to add the geo information to the displayed messages

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2,61 +2,66 @@
...
   "version": "1.0.0",
   "license": "GPL-3.0-or-later",
-  "dependencies": {},
+  "dependencies": {
+    "leaflet": "^1.5.1",
+    "leaflet-draw": "^1.0.4",
+    "react-leaflet": "^2.3.0",
+    "react-leaflet-draw": "^0.19.0"
+  },
   "devDependencies": {
```
Comment (kjoscha) [inline package.json]: @0x46616c6b We wonder which dependencies are supposed to be assigned as devDependencies

Comment (0x46616c6b) [reply package.json]: My understanding is, that for building react apps it don't need to split dependencies between prod and dev. I assume that only imports from the application will bundle in the production javascript.

File: src/api/Message.js [hunk 2]
```diff
@@ -17,10 +17,11 @@ export function getMessages(ticker) {
  * @param {string} text
  * @returns {Promise<Response>}
  */
-export function postMessage(ticker, text) {
+export function postMessage(ticker, text, geoInformation) {
     return Auth.fetch(`${ApiUrl}/admin/tickers/${ticker}/messages`, {
         body: JSON.stringify({
-            text: text
+            text: text,
+            geoInformation: geoInformation
```
Comment (0x46616c6b) [inline src/api/Message.js]: I used underscore notation for keys inside the api.
```suggestion
            geo_information: geoInformation
```

**Comment by 0x46616c6b** [comment]
I checked the PR with the new ticker-api with geo_information support. The new field saved successfully in the database and the message contains the information as well. 

We need to add the geo information, if present, in the message list. The textarea with the geo information from the map should also be hidden before we can merge this PR.

**Comment by 0x46616c6b** [comment]
And another point: If you click on "Show map" you will start with an view of the hole world. In the first iteration we should zoom on germany. Later we can make this dynamically.

**Comment by kjoscha** [comment]
@0x46616c6b please review latest changes. see also ticker-frontend merge request https://github.com/systemli/ticker-frontend/pull/14

We think it is a good way to merge this simple version soon and treat usability improvements in extra merge requests. We have a lot of ideas and gonna put them into issues soon

**Comment by 0x46616c6b** [comment]
Nice! I will check this asap. I'm really happy to see progress and more ideas from your side!

**Pull request merged**